shell/browser/api/electron_api_app.cc

```diff
@@ -513,17 +515,20 @@ int GetPathConstant(const std::string& name) {
 bool NotificationCallbackWrapper(
     const base::RepeatingCallback<
         void(const base::CommandLine& command_line,
-             const base::FilePath& current_directory)>& callback,
+             const base::FilePath& current_directory,
+             const base::span<const uint8_t>& additional_data)>& callback,
     const base::CommandLine& cmd,
-    const base::FilePath& cwd) {
+    const base::FilePath& cwd,
+    const base::span<const uint8_t>& additional_data) {
   // Make sure the callback is called after app gets ready.
   if (Browser::Get()->is_ready()) {
-    callback.Run(cmd, cwd);
+    callback.Run(cmd, cwd, additional_data);
   } else {
     scoped_refptr<base::SingleThreadTaskRunner> task_runner(
         base::ThreadTaskRunnerHandle::Get());
-    task_runner->PostTask(
-        FROM_HERE, base::BindOnce(base::IgnoreResult(callback), cmd, cwd));
+    task_runner->PostTask(FROM_HERE,
```

Comment:
`base::span` is just a view into the allocated memory region, since it does not manage the lifetime we cannot safely pass it around on a different sequence. I would suggest to copy the data into a different container here.